### DISCUSSION TITLE
Create a Case in Amazon using the SP-API

### DISCUSSION THREAD

**Discussion by carmad94** [amzn/selling-partner-api-models#3589] [General] [answered]
Hello, 
Is it possible to create a case in Amazon using the SP-API?
Based on the API documentation, I haven't found any API endpoint that will cater to these use cases.

**Comment by sungolivia** [comment]
I am not sure what you mean by create a case? Do you mean developer support case? If you can expand a bit it would help!

**Reply by carmad94** [reply]
It is more of like an Amazon seller central support case regarding products, listing or inventory

**Reply by rugved1991** [reply, marked as answer]
Hi @carmad94,

Currently it’s not possible to create cases through the api. You will have to log into your seller central account and create a case manually. If you would like to open a case with developer support regarding any SP API issues, please use any one of these links:
Developer Support EU: https://developer.amazonservices.co.uk/support

Developer Support NA: https://developer.amazonservices.com/support

Developer Support FE: https://developer.amazonservices.com.au/support

Best,
Rugved
Solutions Architect, SP API
